docs/backend_architecture/uap/concepts_and_definitions.rst

```diff
@@ -44,13 +42,12 @@ Class A contains two LearnerGroups, Group Q and Group R.
 Membership
 ----------
 
-A ``FacilityUser`` (but not a ``DeviceOwner``) can be marked as a member of a
-``Collection`` through a ``Membership`` object. Being a member of a Collection
-also means being a member of all the Collections above that Collection in the
-hierarchy. Thus, in the illustration below, Alice is directly associated with
-Group Q through a ``Membership`` object, which makes her a member of Group Q.
-As Group Q is contained within Class A, which is contained within Facility X,
-she is also implicitly a member of both those collections.
+A ``FacilityUser`` can be marked as a member of a ``Collection`` through a ``Membership`` object.
+Being a member of a Collection, requires first being a member of
```

Comment:
```suggestion
Being a member of a Collection requires first being a member of
```

Reply:
the capitalization of Collection and Group seems a little odd. Maybe either format words as code if referring to the class (`Collection`) or lower-case it if referring to the concept (collection)